cognite/client/_api/events.py

```diff
@@ -291,7 +292,30 @@ def aggregate(self, filter: Union[EventFilter, Dict] = None) -> List[EventAggreg
                 >>> aggregate_type = c.events.aggregate(filter={"type": "failure"})
         """
 
-        return self._aggregate(filter=filter, cls=EventAggregate)
+        return self._aggregate(filter=filter, cls=AggregateResult)
+
+    def unique_values_aggregate(
```

Comment:
I'd prefer something like `aggregates.unique_values` so it's less messy once we add the rest of the aggregations. Otherwise, I propose `aggregate_unique_values`. Then all aggregations start with `aggregate` which makes it easier to use with autocomplete

Reply:
Ah, we didn't do that for the javascript sdk... https://github.com/cognitedata/cognite-sdk-js/blob/v1/src/resources/events/eventsApi.ts#L74

Reply:
@fredrik955 yeah I know..